From b02a666a64415be98f86c8af69f9ad99789c5193 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 3 Jun 2019 18:18:37 -0600
Subject: [PATCH] manifest: android-9.0.0_r37 -> android-9.0.0_r40

* Also, reference the hw/google/pixel project by named tag as opposed
  to an "anonymous" SHA-1 checksum (no change, same ref)

Change-Id: Ib1ea2a1439a2d8fe95d0871139af6f53ed0f8229
---
 default.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/default.xml b/default.xml
index 5e69a69..6a789b3 100644
--- a/default.xml
+++ b/default.xml
@@ -14,7 +14,7 @@
   <remote  name="aosp"
            fetch="https://android.googlesource.com"
            review="android-review.googlesource.com"
-           revision="refs/tags/android-9.0.0_r37" />
+           revision="refs/tags/android-9.0.0_r40" />
 
   <default revision="refs/heads/lineage-16.0"
            remote="github"
@@ -436,7 +436,7 @@
   <project path="hardware/google/av" name="platform/hardware/google/av" groups="pdk" remote="aosp" />
   <project path="hardware/google/easel" name="platform/hardware/google/easel" groups="pdk,easel" remote="aosp" />
   <project path="hardware/google/interfaces" name="platform/hardware/google/interfaces" groups="pdk" remote="aosp" />
-  <project path="hardware/google/pixel" name="platform/hardware/google/pixel" groups="bonito,crosshatch" remote="aosp" revision="2656ca3347d74d0bde7552fb57a2fd8af89ebb6a" />
+  <project path="hardware/google/pixel" name="platform/hardware/google/pixel" groups="bonito,crosshatch" remote="aosp" revision="refs/tags/android-9.0.0_r42" />
   <project path="hardware/intel/audio_media" name="platform/hardware/intel/audio_media" groups="intel,pdk" remote="aosp" />
   <project path="hardware/intel/bootstub" name="LineageOS/android_hardware_intel_bootstub" groups="intel,pdk" remote="github" />
   <project path="hardware/intel/common/bd_prov" name="platform/hardware/intel/common/bd_prov" groups="intel,pdk" remote="aosp" />
-- 
2.17.1

